pass/pass.go

```diff
@@ -104,11 +104,10 @@ func (p Pass) Delete(serverURL string) error {
 }
 
 func getPassDir() string {
-	passDir := "$HOME/.password-store"
-	if envDir := os.Getenv("PASSWORD_STORE_DIR"); envDir != "" {
-		passDir = envDir
+	if passDir := os.Getenv("PASSWORD_STORE_DIR"); passDir != "" {
+		return passDir
 	}
-	return os.ExpandEnv(passDir)
+	return os.ExpandEnv("$HOME/.password-store")
```

Comment:
I think it would be safer to use [`os.UserHomeDir()`](https://pkg.go.dev/os#UserHomeDir):

```suggestion
	homedir, err := os.UserHomeDir()
	if err != nil {
		return "", err
	}
	return path.join(homedir, ".password-store"), nil
```

WDYT?

Reply:
Heh, yes, I have something like that staged https://github.com/docker/docker-credential-helpers/pull/287#issuecomment-1566196273

> I'll do a follow-up to improve the home-dir lookup (now that we made pass compile for other platforms as well), but making this fix separate first.

Reply:
I actually started with that, and then changed it back to the most minimal changes for this PR 😅